vision/unit_tests/test_safe.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Google Inc.
```

Comment:
High-level

1. If the `safe` module already existed, this test should have also already existed. Transitive test coverage is not the same as writing actual tests
1. If `safe` was small enough to get transitive coverage without every directly testing anything in it, you probably didn't need to give it it's on module
1. :-1: to the module name `safe`, it is unclear